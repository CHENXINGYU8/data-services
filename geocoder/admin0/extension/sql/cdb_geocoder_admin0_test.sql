CREATE EXTENSION postgis;
CREATE EXTENSION schema_triggers;
CREATE EXTENSION plpythonu;
CREATE EXTENSION cartodb;
CREATE EXTENSION cdb_geocoder_admin0;

-- Check that the synonym function is callable, should return NULL
SELECT (admin0_synonym_lookup(Array['United States', 'ESP'])).*;

-- Check that the geocoding function is callable, should return success = false
SELECT (geocode_admin0_polygons(Array['Spain', 'USA', ''])).*;
