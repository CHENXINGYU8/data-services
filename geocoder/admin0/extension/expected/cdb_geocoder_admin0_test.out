CREATE EXTENSION postgis;
CREATE EXTENSION schema_triggers;
CREATE EXTENSION plpythonu;
CREATE EXTENSION cartodb;
CREATE EXTENSION cdb_geocoder_admin0;
-- Check that the synonym function is callable, should return NULL
SELECT (admin0_synonym_lookup(Array['United States', 'ESP'])).*;
       q       | adm0_a3 
---------------+---------
 United States | 
 ESP           | 
(2 rows)

-- Check that the geocoding function is callable, should return success = false
SELECT (geocode_admin0_polygons(Array['Spain', 'USA', ''])).*;
   q   | geom | success 
-------+------+---------
       |      | f
 Spain |      | f
 USA   |      | f
(3 rows)

